src/index.html

```diff
@@ -11,7 +11,189 @@
+          id="email" 
+          name="email" 
+          placeholder="email@example.com" 
+          required
+          >
+        </div>
+        <div class="data">
+          <label for="password">
+            Password:
+          </label>
+          <input 
+          type="password" 
+          id="password" 
+          name="password" 
+          minlength="6" 
+          maxlength="12"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form__group">
+        <legend>An interesting fact about you!</legend>
+        <div>
+          <label for="love_cats">
+            Do you love cats?
+          </label>
+          <input 
+          type="radio" 
+          name="cats" 
+          id="yes"
+          >
+          <label for="yes">
+            Yes
+          </label>
+          <input 
+          type="radio" 
+          name="cats" 
+          id="no"
+          >
+          <label for="no">
+            No
+          </label>
+        </div>
+        <div class="data">
+          <label for="color">
+            What is your favorite color?
+          </label>
+          <input 
+          type="color" 
+          name="color" 
+          id="color"
+          >
+        </div>
+        <div class="data">
+          <label for="time"
+          >What time do you go to bed?
+          </label>
+          <input 
+          type="time" 
+          name="time" 
+          id="time"
+          >
+        </div>
+        <div class="data">
+          <label for="car">
+            What are you favorite brand of cars?:
+          </label>
+          <select 
+          name="car" 
+          id="car" 
+          multiple
+          >
+            <option value="#f01">BMW</option>
+            <option value="#f02">Audi</option>
+            <option value="f#03">Lada</option>
+          </select>
+        </div>
+        <div class="data">
+          <label for="work">
+            How do you rate our work?
+          </label>
+          <input 
+          type="range" 
+          name="work" 
+          id="work" 
+          min="0" 
+          max="25"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form__group">
+        <legend>Additional info:</legend>
+        <div>
+          <label for="comments">
+            Comments:
+          </label>
+          <textarea 
+          name="comments" 
+          id="comments" 
+          cols="20" 
+          rows="2">
+          </textarea>
+        </div>
+        <div class="data">
+          <label for="recommend">Would you recommend us?</label>
+          <select name="recommend" id="recommend">
+            <option value="#f01">Yes</option>
+            <option value="#f02">No</option>
+          </select>
+        </div>
+      </fieldset>
+      <button 
+      class="form__group"
+      type="submit"
+      >
+        Submit
+      </button>
+    </form>
     <script type="text/javascript" src="./main.js"></script>
   </body>
-</html>
+</html>
```

Comment:
add empty line in the end

Reply:
Hi! I have fixed the tags indentation and added the empty line at the end.
Please check and let me know if everything is done correctly.
TY